### ISSUE TITLE
Ensure single submission Enketo forms do not reload after one submission

### ISSUE THREAD

**Issue by bkimondiu** [onaio/onadata#1710]
### Environment (local, stage, preview, production)
Production

### Expected behavior
Creating a single submission link similar to `https://enketo.ona.io/single/::ejEbVdou` should allow users to submit a form only once. The form should not load for data collection again when the link is opened. 

### Actual behavior
Creating a single submission link similar to `https://enketo.ona.io/single/::ejEbVdou`, allows users to make a submission but opens again when a user clicks the link.

### Steps to reproduce the behavior
1. Create a form and upload it to Ona
2. Click the Webform button to launch the form on Enketo
3. Change the form's Enketo link from e.g. `https://enketo.ona.io/x/#ejEbVdou` to `https://enketo.ona.io/single/::ejEbVdou`
4. Copy the new link to a new tab and submit data on it
5. Copy the link again to a new tab

**Comment by lincmba** [comment]
Enketo single submission is not enforced by Onadata but by Enketo. Also, it's harder to enforce this with public forms as they can be filled by users outside Ona without any form of authentication.

**Comment by moshthepitt** [comment]
@lincmba @bkimondiu  in your view, is a wontfix acceptable for this issue?

**Comment by lincmba** [comment]
@moshthepitt Yes. Onadata doesn't have this feature. Single submit is only enforced by Enketo as explained here https://blog.enketo.org/single-submission-surveys/ and Enketo issues should be raised here https://github.com/enketo/enketo-express/issues.

Should we choose to work on enforcing this, then this should be a feature request.

**Comment by moshthepitt** [comment]
Cool, thanks @lincmba
